app/components/UserGrid/index.tsx

```diff
@@ -26,17 +28,23 @@ const UserGrid = ({
       }px, 1fr))`,
       gridTemplateRows: users.length === 0 ? 0 : size + padding,
       overflow: 'hidden',
-      minHeight: minRows * size + (minRows - 1) * padding,
-      maxHeight: maxRows * size + (maxRows - 1) * padding,
+      minHeight: minRows * size + (minRows - 1) * padding + 3.3,
+      maxHeight: maxRows * size + (maxRows - 1) * padding + 3.3,
```

Comment:
`3.3` pixels are added here and in `UserGrid.css` so that, once people start registering, the skeleton and profile pictures will be aligned. This is because the images contain margins. It's not very clean, but it's better than nothing, and I've spent way too many hours trying to find a better solution ..